draft-ietf-httpbis-digest-headers.md

```diff
@@ -1382,7 +1386,7 @@ _RFC Editor: Please remove this section before publication._
 ## Since draft-ietf-httpbis-digest-headers-01
 {:numbered="false"}
 
-* Digest of error responses is computed on the error representation-data #1004
+* Digest of error responses is computed on the error representation data #1004
```

Comment:
I don't think we need to worry about updating terms in the changelog. That can quickly become an impossible task.

Reply:
it was just a s//g ;)